components/scream/src/mct_coupling/scream_cpl_indices.F90

```diff
@@ -47,183 +39,69 @@ subroutine scream_set_cpl_indices (x2a, a2x)
     !
     integer :: i,idx
 
+    ! total number of import/export var in data
+    num_cpl_imports = size(x2a%rAttr,1)
+    num_cpl_exports = size(a2x%rAttr,1)
```

Comment:
Main change: these are no longer explicitly defined. We now query the x2a/a2x arrays.

Reply:
The main changes are in this file, but the diff is hard to read.. May be easier to look at the new file as a whole.